app/javascript/articles/components/Meta.jsx

```diff
@@ -80,7 +80,7 @@ export const Meta = ({ article, organization }) => {
         <a href={article.path} className="crayons-story__tertiary fs-xs">
           <PublishDate
             readablePublishDate={article.readable_publish_date}
-            publishedTimestap={article.published_timestamp}
+            publishedTimestamp={article.published_timestamp}
```

Comment:
I wonder why the proptypes didn't complain about this. 🤔

Reply:
I don't think it gives warnings if you pass an 'extra' prop... and the `publishedTimestamp` prop isn't marked as required (maybe it should be though?)

Reply:
Ahh didn't realize it was an extra prop. If it's always there, I'd add it as a required prop.